phishtray/base.py

```diff
@@ -49,15 +54,6 @@ def filter_by_org_public(self, user):
 
         return self
 
-    def filter_by_user(self, user):
-        if not user or not isinstance(user, User):
-            return self.none()
-
-        if user.is_superuser:
-            return self
-
-        return self.filter(id=user.organization_id)
-
```

Comment:
This is referred in the Slack comment please.

Reply:
resolved